app/main.py

```diff
@@ -2,0 +1,26 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    route = command[2].split("/")
+    if command[0] != "mv":
+        return
```

Comment:
You're checking if the command is 'mv'. This is a good check, but it would be better to do this before you split the command into its components. That way, you won't do unnecessary work for commands that aren't 'mv'.